corehq/apps/sms/forms.py

```diff
@@ -69,6 +69,18 @@
 TIME_AFTER = "AFTER"
 TIME_BETWEEN = "BETWEEN"
 
+WELCOME_RECIPIENT_NONE = 'NONE'
+WELCOME_RECIPIENT_CASE = 'CASE'
+WELCOME_RECIPIENT_MOBILE_WORKER = 'MOBILE_WORKER'
+WELCOME_RECIPIENT_ALL = 'ALL'
+
+WELCOME_RECIPIENT_CHOICES = (
+    (WELCOME_RECIPIENT_NONE, ugettext_lazy('Nobody')),
```

Comment:
@esoergel were we going to move away from `ugettext_lazy` in favor of being explicit about `ugettext_noop` and `uggettext`?